### ISSUE TITLE
fixed-small-slow-III container needs to show standfirst when image is missing on smaller breakpoints

### ISSUE THREAD

**Issue by michaelwmcnamara** [guardian/frontend#6649]
fixed-small-slow-III container needs to show standfirst when image is missing
It does this at the moment on breakpoints wider than 980' but we need to do this for smaller breakpoints ie the ones that lie between width 740' and width 980'  
(eg ipad mini in portrait mode but not iphone)

Discussed with Sam, this could be part of a more general issue where we need to detect when an image is not shown and display the standfirst. 

example:

http://www.theguardian.com/us/sustainable-business shown on ipad mini in portrait mode:
![ipadminiport](https://cloud.githubusercontent.com/assets/7239344/4721968/5301eeaa-593c-11e4-892e-841cc2cf29d9.PNG)

same page in landscape (slightly wider breakpoint - standfirst makes it much nicer):
![ipmadminils](https://cloud.githubusercontent.com/assets/7239344/4721984/86ebdc6c-593c-11e4-8833-ac055f7c0fe4.PNG)

Basically - can we make portrait mode look more like landscape mode?

**Comment by johnduffell** [comment]
closing this as it hasn't been fixed so far and probably won't be fixed in the near future
